frontend/src/metabase/home/components/Activity.jsx

```diff
@@ -89,43 +89,43 @@ export default class Activity extends Component {
+                    description.summary = (<span>{t`added a question to the dashboard`} - <Link to={Urls.dashboard(item.model_id)} data-metabase-event={"Activity Feed;Header Clicked;Dashboard -> "+item.topic} className="link text-dark">{item.details.name}</Link></span>);
                 } else {
-                    description.summary = (<span>added a question to the dashboard - <span className="text-dark">{item.details.name}</span></span>);
+                    description.summary = (<span>{t`added a question to the dashboard`} - <span className="text-dark">{item.details.name}</span></span>);
                 }
                 break;
             case "dashboard-remove-cards":
                 if(item.model_exists) {
-                    description.summary = (<span>removed a question from the dashboard - <Link to={Urls.dashboard(item.model_id)} data-metabase-event={"Activity Feed;Header Clicked;Dashboard -> "+item.topic} className="link text-dark">{item.details.name}</Link></span>);
+                    description.summary = (<span>{t`removed a question from the dashboard`} - <Link to={Urls.dashboard(item.model_id)} data-metabase-event={"Activity Feed;Header Clicked;Dashboard -> "+item.topic} className="link text-dark">{item.details.name}</Link></span>);
                 } else {
-                    description.summary = (<span>removed a question from the dashboard - <span className="text-dark">{item.details.name}</span></span>);
+                    description.summary = (<span>{t`removed a question from the dashboard`} - <span className="text-dark">{item.details.name}</span></span>);
                 }
                 break;
             case "database-sync":
                 // NOTE: this is a relic from the very early days of the activity feed when we accidentally didn't
                 //       capture the name/description/engine of a Database properly in the details and so it was
                 //       possible for a database to be deleted and we'd lose any way of knowing what it's name was :(
-                const oldName = (item.database && 'name' in item.database) ? item.database.name : "Unknown";
+                const oldName = (item.database && 'name' in item.database) ? item.database.name : t`Unknown`;
                 if(item.details.name) {
-                    description.summary = (<span>received the latest data from <span className="text-dark">{item.details.name}</span></span>);
+                    description.summary = (<span>{t`received the latest data from`} <span className="text-dark">{item.details.name}</span></span>);
                 } else {
-                    description.summary = (<span>received the latest data from <span className="text-dark">{oldName}</span></span>);
+                    description.summary = (<span>{t`received the latest data from`} <span className="text-dark">{oldName}</span></span>);
```

Comment:
Not related to i18n but I might break some of these summaries onto new lines to make them a bit easier to parse.